lib/irb/default_commands.rb

```diff
@@ -236,6 +229,27 @@ def self.load_command(command)
       nil
     end
 
+    def self.install_helper_methods
+      HelperMethod.helper_methods.each do |name, helper_method_class|
+        define_method name do |*args, **opts, &block|
+          verbose, $VERBOSE = $VERBOSE, nil
+          helper_ivar = "@_helper_method_#{name}".to_sym
+          helper = instance_variable_get(helper_ivar)
+
+          unless helper
+            helper = helper_method_class.new
+            instance_variable_set(helper_ivar, helper)
+          end
+
+          helper.execute(*args, **opts, &block)
```

Comment:
Helper method is an instance method of `workspace.main`. If we pass `self` to execute, perhaps helper_method can do more.
```ruby
helper.execute(self, *args, **opts, &block)
```
We can get a similar value by `IRB.CurrentContext.workspace.main` except for this case: `$a=self` `chws ""` `$a.conf` (`main==""` but `self==$a`), so we don't need to pass it?
What do you think?